<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QMainWindow" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>817</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Narrow</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
color:white;

border:none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="Container">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignRight|Qt::AlignVCenter">
        <widget class="QFrame" name="SelectCategory">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;


QCheckBox{
background-color:white;
color:black;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color:black;
</string>
            </property>
            <property name="text">
             <string>Child Category</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color:black;
</string>
            </property>
            <property name="text">
             <string>Header Catergory</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="ChildCheckbox">
            <property name="minimumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>CheckBox</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="ParentCheckbox">
            <property name="minimumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QFrame" name="InfoContainer">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="Image">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Resources.qrc">:/images/EnterDetailImage.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Info">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Rockwell Extra Bold</family>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 800 15pt &quot;Rockwell Extra Bold&quot;;
text-decoration:underline;</string>
            </property>
            <property name="text">
             <string>Enter Details:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QFrame" name="Details">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
background-color: gray;
font: 12pt &quot;Maiandra GD&quot;;
border-radius: 7px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
               </property>
               <property name="text">
                <string>NAME:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ReadName">
               <property name="styleSheet">
                <string notr="true">background-color: gray;
font: 12pt &quot;Maiandra GD&quot;;
border-radius:7px;




</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
               </property>
               <property name="text">
                <string>CODE:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ReadCode">
               <property name="styleSheet">
                <string notr="true">background-color: gray;
font: 12pt &quot;Maiandra GD&quot;;
border-radius: 7px;



</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
               </property>
               <property name="text">
                <string>TYPE:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ReadType">
               <property name="minimumSize">
                <size>
                 <width>340</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>ASSETS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>LIABILITIES</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>EQUITY</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="BalanceFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
               </property>
               <property name="text">
                <string>OPENING BALANCE:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ReadBalance">
               <property name="styleSheet">
                <string notr="true">background-color: gray;
font: 12pt &quot;Maiandra GD&quot;;
border-radius: 7px;




</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="ParentFrame">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
               </property>
               <property name="text">
                <string>PARENT HEADER:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ReadHeader">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Notes">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(164, 170, 158)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_9">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 800 15pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(18, 18, 18);
padding: 15;</string>
            </property>
            <property name="text">
             <string>NOTES:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="minimumSize">
                <size>
                 <width>500</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: gray;

font: 13pt &quot;Maiandra GD&quot;;
border-radius: 7px;



</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QFrame" name="frame_9">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	border:2px solid black;
	border-radius:5px;
	background-color:grey;
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="Cancel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;
</string>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Save">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 800 10pt &quot;Rockwell Extra Bold&quot;;</string>
               </property>
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
